tools/submission/submission_checker.py

```diff
@@ -2381,6 +2413,39 @@ def check_measurement_dir(
             if not os.path.exists(os.path.dirname(code_dir)):
                 log.error("%s is missing code_dir %s", fname, code_dir)
                 is_valid = False
+  
+        # Check equal issue mode        
+        if "mlperf.conf" in files and config.requires_equal_issue(model, division):
+            with open(f"{measurement_dir}/mlperf.conf") as f:
+                lines = f.readlines()
+                equal_issue_used = False
+                conf_ref_model = model.replace("-99", "").replace("-99.9", "")
+                for line in lines:
+                    line = line.replace(" ", "").replace("\n", "")
+                    if line.startswith("#"):
+                        continue
+                    elif line == "":
+                        continue
+                    else:
+                        key, val = line.split("=")
+                        key.replace(" ", "")
+                        val.replace(" ", "")
+                        conf_model, conf_scenario, conf_key = key.split(".")
+                        if (
+                            (conf_key == "sample_concatenate_permutation") and 
+                            ((conf_model == conf_ref_model) or conf_model == "*") and
+                            ((conf_scenario == scenario) or conf_scenario == "*")
+                        ):
+                            if val.isnumeric():
+                                val = int(val)
+                                equal_issue_used = (val == 1)
+                                break
+                if not equal_issue_used:
+                    log.error(
+                        "%s requires equal issue mode (sample_concatenate_permutation), expected=true, found=%s",
```

Comment:
is_valid =False

Should also be added here

Reply:
Added it